cloudbridge/cloud/providers/gce/services.py

```diff
@@ -402,10 +405,133 @@ def list(self, limit=None, marker=None):
                                      limit=limit, marker=marker)
 
 
+class GCEInstanceService(BaseInstanceService):
+
+    def __init__(self, provider):
+        super(GCEInstanceService, self).__init__(provider)
+
+    def create(self, name, image, instance_type, zone=None,
+               key_pair=None, security_groups=None, user_data=None,
+               launch_config=None,
+               **kwargs):
+        """
+        Creates a new virtual machine instance.
+        """
+        if zone is None:
+            zone = self.provider.default_zone
+        if launch_config is None:
```

Comment:
[Style guide](https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#True/False_evaluations) suggests using implicit True/False.

So this should be:

    if not launch_config:

Similar comment on the "if zone is None" check above. But I would defer to the project style convention.